css/ndt-test.css

```diff
@@ -0,0 +1,337 @@
+  display: block;
+}
+
+#welcome .info {
+  padding-top: 10px;
+  text-transform: none;
+  font: 13px/18px verdana, helvetica, arial, sans-serif;
+}
+#welcome .info .logo {
+  margin-bottom: 30px;
+}
+#welcome .info p {
+  margin-bottom: 20px;
+}
+#welcome .info a {
+  color: white;
+  font-weight: bold;
+}
+
+
+/* Test */
+
+#test .locations {
+  width: 640px;
+  position: absolute;
+  left: 40px;
+  top: 400px;
+  /*color: white;*/
+  font-size: 32px;
+  line-height: 32px;
+}
+#test .location {
+  position: absolute;
+  top: 0;
+}
+#test .location.local {
+  right: 0;
+  text-align: right;
+}
+#test .location.remote {
+  left: 0;
+  text-align: left;
+}
+#test .location .address {
+  margin-top: 10px;
+  color: black;
+  font: bold 13px/13px verdana, helvetica, arial, sans-serif;
+}
+
+#uploadGauge {
+  position: absolute;
+  left: 40px;
+  top: 120px;
+  overflow: hidden;
+}
+
+#downloadGauge {
+  position: absolute;
+  left: 380px;
+  top: 120px;
+  overflow: hidden;
+}
+
+#rtt {
+  position: absolute;
+  left: 320px;
+  top: 320px;
+  color: white;
+  font: bold 20px/20px verdana, helvetica, arial, sans-serif;
+}
+
+#rttValue {
+  position: relative;
+  text-align: center;   
+  left: -30px;
+  width: 100px;
+  color: black;
+}
+
+/* Test results */
+
+#results {
+  height: 400px;
+}
+#result-pages {
+  width: 640px;
+  height: 280px;
+  position: absolute;
+  left: 40px;
+  top: 100px;
+}
+#result-pages .result-page {
+  width: 640px;
+  height: 280px;
+  position: absolute;
+  left: 0;
+  top: 0;
+  display: none;
+}
+#results.summary #result-pages .result-page.summary,
+#results.details #result-pages .result-page.details,
+#results.advanced #result-pages .result-page.advanced {
+  display: block;
+}
+#result-pages .details,
+#result-pages .advanced {
+  overflow: auto; 
+  text-transform: none;
+  font: 13px/18px verdana, arial, helvetica, sans-serif;
+}
+#result-pages .advanced {
+  white-space: pre;
+}
+#results .result {
+  width: 268px;
+  height: 180px;
+  position: absolute;
+  top: 0px;
+  padding: 20px;
+  border: 1px solid #ccc;
+}
+#results .result .number {
+  font-size: 100px;
+  line-height: 128px;
+}
+#results .result .units {
+  font-size: 48px;
+  text-transform: none;
+}
+#results .upload {
+  left: 0;
+}
+#results .download {
+  right: 0;
+}
+#results .other {
+  position: absolute;
+  top: 240px;
+  text-transform: none;
+  font: 13px/18px verdana, arial, helvetica, sans-serif;
+}
+#results .controls {
+  width: 320px;
+  left: 360px;
+  text-align: right;
+}
+#results .branding {
+  position: absolute;
+  width: 320px;
+  left: 40px;
+  bottom: 24px;
+  text-transform: none;
+  font: 11px/16px verdana, arial, helvetica, sans-serif;
+}
+#results .branding a {
+  color: black;
+}
+
+
+#results .view-selector {
+  position: absolute;
+  right: 40px;
+  top: 60px;
+  font-size: 18px;
+  line-height: 18px;
+}
+#results .view-selector .summary {
+  border-width: 2px 1px 2px 2px;
+  padding-right: 15px;
```

Comment:
Should just be `padding`
https://google.github.io/styleguide/htmlcssguide.xml?showone=Shorthand_Properties#Shorthand_Properties

(ditto for 318-319, 326)